server/svix-server/src/v1/endpoints/auth.rs

```diff
@@ -16,11 +17,24 @@ pub struct DashboardAccessOut {
     pub token: String,
 }
 
+#[derive(Deserialize, Serialize, Validate, JsonSchema)]
+pub struct DashboardAccessIn {
+    #[validate]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub feature_flags: Option<EventTypeFeatureFlagSet>,
+}
+
 async fn dashboard_access(
     State(AppState { cfg, .. }): State<AppState>,
     permissions::OrganizationWithApplication { app }: permissions::OrganizationWithApplication,
+    ValidatedJson(json): ValidatedJson<DashboardAccessIn>,
```

Comment:
```suggestion
    ValidatedJson(data): ValidatedJson<DashboardAccessIn>,
```
I think we call it `data` everywhere else.